src/azure-cli-core/azure/cli/core/commands/parameters.py

```diff
@@ -28,28 +31,16 @@ def get_subscription_locations(cli_ctx):
     return list(subscription_client.subscriptions.list_locations(subscription_id))
 
 
-@Completer
+@func_completer_wrapper
```

Comment:
Won't this break auto-complete? as `func_completer_wrapper`'s implementation is empty.

https://github.com/Azure/azure-cli/blob/607071cfe91d1070f50e041f33387bc637f5e748/src/azure-cli-core/azure/cli/core/translator/_decorators.py#L32-L34

Reply:
Nice catch!